app/src/shared/components/Input/CoreText.jsx

```diff
@@ -0,0 +1,44 @@
+// @flow
```

Comment:
This component is used all over the core now. Replaces both `TextField` and `TextInput`.